Runtime/Native/Android/NativeClient.cs

```diff
@@ -504,11 +502,13 @@ public override void Disable()
             {
                 _anrWatcher.Call("stopMonitoring");
                 _anrWatcher.Dispose();
+                _anrWatcher = null;
```

Comment:
Does the method `Disable` disable this client permanently? Is there a way to re-enable it?

Reply:
Right now this isn't our goal but it might be in the future. Right now this code should handle the situation when someone calls Dispose method directly via Unity API.